test/jest-utils.js

```diff
@@ -0,0 +1,18 @@
+const baseUrl = port => {
+  if (process.env.DEMO_APPS_DOMAIN === 'localhost') {
+    const url = new URL('https://localhost');
```

Comment:
```suggestion
    const url = new URL('http://localhost');
```
This might resolve the test failures.

Reply:
🤫